build.sbt

```
@@ -463,7 +463,11 @@ lazy val docs = project // new documentation project
     scalacOptions ++= Seq(
       "-language:reflectiveCalls",
       "-language:implicitConversions",
-      "-Wconf:msg=firrtl:s,cat=other-implicit-type:s"
+      "-Wconf:msg=firrtl:s,cat=other-implicit-type:s",
```

FWIW, I think this suppression of `other-implicit-type` may be the requirement that public implicits have a type annotation, so we may be able to remove this warning waiver as part of this change.
```suggestion
      "-Wconf:msg=firrtl:s",
```